src/mrinufft/operators/autodiff.py

```diff
@@ -20,14 +20,20 @@ class _NUFFT_OP(torch.autograd.Function):
     @staticmethod
     def forward(ctx, x, traj, nufft_op):
         """Forward image -> k-space."""
-        ctx.save_for_backward(x, traj)
+        if nufft_op._grad_wrt_data and not nufft_op._grad_wrt_traj:
```

Comment:
This seems different, can you please document this update also in the PR ?

Reply:
I think thats for avoiding saving too many tensors. @AsmaTANABEN comments?

Reply:
Yes, i think, she needed this also, we can have it in same PR surely, but lets add that in description and/or title ?

Reply:
yeah it's about memory saving +  if differentiation is not required with respect to traj, it could be a non-tensor variable, which won't cause an error with the ctx.save_for_backward() function.